samples/bitwise.br

```diff
@@ -0,0 +1,5 @@
+let x = 5 & 1;
```

Comment:
instead of adding a sample, add a test or two in the tests package

Reply:
done